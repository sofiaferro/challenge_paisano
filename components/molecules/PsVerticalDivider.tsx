import React from 'react';

import styled from 'styled-components';

import Image from 'next/image';

import dividerVertical from '@/images/divider-vertical.svg';

// style
const Container = styled.div`
  padding: 1em;
`;

const PsHorizontalDivider = () => {
  return (
    <Container>
      <Image src={dividerVertical} alt={'Vertical divider'} />
    </Container>
  );
};

export default PsHorizontalDivider;
