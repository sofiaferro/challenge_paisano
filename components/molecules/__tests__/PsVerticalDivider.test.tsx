import React from 'react';
import renderer from 'react-test-renderer';

import { describe, expect, it } from '@jest/globals';

import PsVerticalDivider from '../PsVerticalDivider';

describe('PsVerticalDivider', () => {
  it('renders the component', () => {
    const tree = renderer.create(<PsVerticalDivider />);
    expect(tree).toMatchSnapshot();
  });
});
