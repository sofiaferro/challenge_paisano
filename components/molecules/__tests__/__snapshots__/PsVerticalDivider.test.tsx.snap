// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PsVerticalDivider renders the component 1`] = `
<div
  className="sc-b06b11df-0 iXUNOy"
>
  <img
    alt="Vertical divider"
    data-nimg="1"
    decoding="async"
    height={40}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
    srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
    style={
      {
        "color": "transparent",
      }
    }
    width={40}
  />
</div>
`;
